docs/public-networks/how-to/troubleshoot/peering.md

```diff
@@ -10,6 +10,8 @@ tags:
 
 Many factors can affect the ability of your node to find and maintain peers. Your network router, machine environment, and node configuration are all important. If you have peering issues, start by [configuring your ports](../connect/configure-ports.md) and [managing peers](../connect/manage-peers.md).
 
+If your nodes are running in AWS, check you have appropriate `SecurityGroups` to allow access to the required ports. See documentation on [configuring ports](../connect/configure-ports.md) for more information. 
+
```

Comment:
```suggestion
```

You don't need this here since the sentence is duplicated in "Configure ports," which is already linked to in the preceding paragraph.

Reply:
Ok. I removed it. I was thinking if user landed on the troubleshooting page before the configure ports, they might not know where to go for that info.

Reply:
Yes but this sentence doesn't apply to all users, it's more of a side note, so should not appear in the text of an intro paragraph. Also no need to link to Configure ports multiple times in a row.